lib/orderbook/OrderBookRepository.ts

```diff
@@ -73,3 +78,4 @@ class OrderbookRepository {
 }
 
 export default OrderbookRepository;
+export { Orders };
```

Comment:
I assume the dependency graph will support it. 

It’s a type which is relevant only for this class (OrderBookRepository), so as always it’s easier to be consumed together with it, and in this case I couldn’t predict a circular dependency situation, because these who consume this class (app-level stuff) will never be consumed by it. 

So basically, the db layer is still low enough to be able to export types.
We can argue that it’s better to put all type definitions in the `types` folder, but i’m not sure about it.